src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
+Microsoft.AnalysisServices;microsoft.analysisservices/servers;Azure Analysis Services;Analytics;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.ApiManagement;microsoft.apimanagement/service;API Management;Web;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.apimanagement/service;API Management;Web;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.App;microsoft.app/containerapps;Azure Container Apps;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.App;microsoft.app/managedenvironments;Azure Container Apps;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.appcenter;microsoft.appcenter/accounts;App Center;Mobile;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.AppConfiguration;microsoft.appconfiguration/configurationstores;App Configuration;Developer Tools;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.appconfiguration/configurationstores;App Configuration;Developer Tools;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AppPlatform;microsoft.appplatform/spring;Azure Spring Apps;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.automation;microsoft.automation/automationaccounts;Azure Automation;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AutonomousDevelopmentPlatform;microsoft.autonomousdevelopmentplatform/workspaces;Microsoft Autonomous Systems;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.autonomousdevelopmentplatform/workspaces;Microsoft Autonomous Systems;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AVS;microsoft.avs/privateclouds;Azure VMware Solution;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.AzureActiveDirectory;microsoft.azureactivedirectory/b2cdirectories;Azure Active Directory External Identities;Identity;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AzureActiveDirectory;microsoft.azureactivedirectory/guestusages;Azure Active Directory External Identities;Identity;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AzureArcData;microsoft.azurearcdata/postgresinstances;Azure Arc;Multicloud;Microsoft;Cloud Provider;Both;SaaS
+Microsoft.AzureArcData;microsoft.azurearcdata/sqlmanagedinstances;Azure Arc;Multicloud;Microsoft;Cloud Provider;Both;SaaS
+Microsoft.AzureArcData;microsoft.azurearcdata/sqlserverinstances;Azure Arc;Multicloud;Microsoft;Cloud Provider;On-premises;SaaS
+Microsoft.AzureStackHCI;microsoft.azurestackhci/clusters;Azure Stack HCI;Multicloud;Microsoft;Cloud Provider;On-premises;IaaS
+Microsoft.BareMetalInfrastructure;microsoft.baremetalinfrastructure/baremetalstorageinstances;BareMetal Infrastructure;Compute;Microsoft;Cloud Provider;Both;IaaS
+Microsoft.Batch;microsoft.batch/batchaccounts;Batch;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Bing;microsoft.bing/accounts;Microsoft Bing;Web;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.BotService;microsoft.botservice/botservices;Azure AI Bot Service;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;SaaS

```

Comment:
Should this be PaaS since it's a dev platform? 🤔

Reply:
Corrected